sonic_installer/common.py

```diff
@@ -41,16 +41,20 @@ def run_command(command, stdout=subprocess.PIPE, env=None, shell=False):
         sys.exit(proc.returncode)
 
 # Run bash command and return output, raise if it fails
-def run_command_or_raise(argv, raise_exception=True):
+def run_command_or_raise(argv, raise_exception=True, nocapture=False):
```

Comment:
>nocapture=False [](http://example.com/codeflow?start=53&length=15)

To help read, could you use positive names? For example `capture=True`

Reply:
@qiluo-msft Done